packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Funding/Amount/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { connect } from 'react-redux'
+import { IconChevronDown } from '@blockchain-com/icons'
+import styled from 'styled-components'
+
+import { getRatesSelector } from '@core/redux/data/misc/selectors'
+import { Text } from 'blockchain-info-components'
+
+import { Continue, FundingHeading, ListItemContent } from '../SelectAccount'
+
+const AmountListItem = styled(ListItemContent)`
+  margin: 30px 0;
+`
+
+const AmountInputWrapper = styled.div`
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+`
+
+const AmountInput = styled.input`
+  width: 140px;
+  background: transparent;
+  color: ${(props) => props.theme.white};
+  border: none;
+  font-size: 40px;
+  font-weight: 600;
+  line-height: 50px;
+  caret-color: ${(props) => props.theme.blue600};
+  text-align: center;
+  &:focus {
+    outline: none;
+  }
+  &::-webkit-outer-spin-button,
+  &::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+  }
+`
+const AmountCurrency = styled.span`
+  position: absolute;
+  right: 20px;
+  width: 45px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 21px;
+  cursor: pointer;
+  white-space: nowrap;
+  text-transform: uppercase;
+`
+const ExchangedValue = styled.div`
+  margin-bottom: 90px;
+  text-transform: uppercase;
+  text-align: center;
+  color: ${(props) => props.theme.grey400};
+`
+class CurrencyAmounts {
+  public eth: number
+
+  public usd: number
+
+  constructor(eth = 0, usd = 0) {
```

Comment:
constructor(public eth: number = 0, public usd: number = 0)

Reply:
Cant, because of linter